javascript/javascript3/week3/delete.js

```diff
@@ -0,0 +1,11 @@
+//Item array removal
+const names = ['Peter', 'Ahmad', 'Yana', 'Kristina', 'Rasmus', 'Samuel', 'Katrine', 'Tala'];
+const nameToRemove = 'Ahmad';
+
+for (var i = 0; i < names.length; i++) {
```

Comment:
This will work. Also consider using `indexOf`, then you can avoid using the for loop and it's much shorter